### DISCUSSION TITLE
How do I import wsse?

### DISCUSSION THREAD

**Discussion by ru88s** [freshframework/fresh#775] [Q&A] [answered]
How do I import wsse?
I want to use wsse to retrieve data from the AtomPub API.

https://www.npmjs.com/package/wsse

Deno 1.25 seems to support npm import, but
`deno task start --unstable`

I run the task, I get the following error message
```
[The --unstable flag must be provided.
The --unstable flag must be provided.](error: Uncaught (in promise) TypeError: Unstable use of npm specifiers. The --unstable flag must be provided.)
```
What should I do?

https://deno.com/blog/v1.25

**Comment by irvanmalik48** [comment, marked as answer]
Try importing from esm.sh : <https://esm.sh/wsse@5.1.0?target=deno>

**Reply by ru88s** [reply]
```js
import wsse from "https://esm.sh/wsse@5.1.0?target=deno";
```

It worked!
Thank you very much.
